### DISCUSSION TITLE
How do I open more ports for the main EKS containers?

### DISCUSSION THREAD

**Discussion by rhoboat** [gruntwork-io/knowledge-base#174] [Help] [answered]
Howdy 🤠 We are using the `k8s-service` module for launching apps into EKS. My question is, how do we define more than one open port for the container? I have been able to do this for our sidecar containers but am unsure of the syntax/variables needed to do this for the main container in the deployment.

**Comment by autero1** [comment]
While the underlying [Helm chart](https://github.com/gruntwork-io/helm-kubernetes-services/tree/master/charts/k8s-service) supports multiple container ports, the [`k8s-service`](https://github.com/gruntwork-io/terraform-aws-service-catalog/tree/master/modules/services/k8s-service) does not have the ability to configure those. I've created an issue to add support for configuring additional container ports: https://github.com/gruntwork-io/terraform-aws-service-catalog/issues/1189

**Comment by autero1** [comment, marked as answer]
Support for configuring multiple ports added in [Service Catalog Release `v0.85.4`](https://github.com/gruntwork-io/terraform-aws-service-catalog/releases/tag/v0.85.4).
